cogs/utils/db.py

```diff
@@ -172,6 +172,21 @@ async def set_member(guildID, memberID, option, value):
     await conn.close()
 
 
+# Returns members who have birthday on a given date
+async def get_members_bday_equals_date(date):
+    conn = await db_connect()
+    c = await conn.cursor()
+    sql = f"""
+    SELECT MIN(member_id) AS member_id, *
+    FROM members
+    WHERE birthday LIKE '%{date.strftime('%m')}-{date.strftime('%d')}%'
```

Comment:
`birthday` in the bot.db will always be stored in the format of `%Y-%m-%d` (the bot throws an error to the user if it fails to parse their input into that format), so this extra check is probably unnecessary. If you can think of a situation where this might still be necessary though, please let me know!

Reply:
Have tried multiple times giving the wrong format (for an example 2019-12-1) and the bot doesn't throw any error. That's why I did it that way.

Reply:
![image](https://user-images.githubusercontent.com/61143147/100801668-99498300-3428-11eb-8117-27c72c7e28cb.png)

Reply:
Huh, that's interesting. `2019-12-1` is technically a valid date if `.strptime()` is able to interpret `1` as `01`, but I'm not sure if that's how it works. What other values did you test? If you check the `bot.db`, does the date `2019-12-1` get properly converted to `2019-12-01`? Perhaps `.strptime()` doesn't care about the formatting difference of `1` vs `01`?

Reply:
It's saving as 1 not 01.

Reply:
`2019-1-01` and `2019-1-1` get a pass and are not in a valid format.

Reply:
Does that fail to parse when the call does just the check for `LIKE date.month-date.day`?

Reply:
It fails if the `date.month` or `date.day` are single digits and the stored date is in a valid format.

Reply:
Huh, okay, that's an issue with the check to store data then. Thanks for the effort in double-checking. You can leave this section of code as-is, I'll create a bug report for the issue and get that fixed separately from this